tests/hw-accel/kmm/modules/tests/dra-upgrade-test.go

```
@@ -0,0 +1,345 @@
+					5*time.Minute, GeneralConfig.WorkerLabelMap)
+				Expect(err).ToNot(HaveOccurred(), "error waiting for module deployment")
+
+				By("Await DRA deployment")
+
+				err = await.DRADeployment(APIClient, moduleName, nSpace,
+					3*time.Minute, GeneralConfig.WorkerLabelMap)
+				Expect(err).ToNot(HaveOccurred(), "error waiting for DRA deployment")
+
+				By("Verify DRA DaemonSet exists and record its name")
+
+				ds, err := get.DRADaemonSet(APIClient, moduleName, nSpace)
```

golangci-lint (2.12.2)

[error] variable name 'ds' is too short for the scope of its usage

(varnamelen)